ConnectionFactory.php

```diff
@@ -78,7 +79,9 @@ public function createConnection(array $params, ?Configuration $config = null, ?
             }
 
             if (! isset($params['charset'])) {
-                if ($driver instanceof AbstractMySQLDriver) {
+                if ($driver->getDatabasePlatform() instanceof AbstractMySQLPlatform
```

Comment:
The resulting Psalm error will have to be ignored through configuration I think.

Reply:
I'm not familiar with psalm do I need to ignore the whole file or is there another way?

Reply:
okay seems `@psalm-suppress UndefinedClass` does the job